### PR TITLE
Add Seaweed FS to devspace/k8s

### PR DESCRIPTION
**Pull request by yashgorana**
## Description

- Closes https://github.com/OpenMined/Heartbeat/issues/752

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: packages/grid/devspace.yaml [hunk 1]
```diff
@@ -218,6 +224,58 @@ deployments:
...
+          - name: seaweedfs-data
+            size: "5Gi"
+          - name: seaweedfs-conf
+            configMap:
+              name: seaweedfs-conf
+        service:
+          name: seaweedfs
+          ports:
+            - port: "8888" # filer
+            - port: "8333" # S3
```
Comment (yashgorana) [inline packages/grid/devspace.yaml:272]: Please let me know if we need to expose these ports, else I can move them under "dev"

**Comment by yashgorana** [comment]
I'll test this out before I remove the [WIP] tag

**Comment by yashgorana** [comment]
@madhavajay @rasswanth-s tested the kubernetes deployments. I had to tweak `tox.ini` & `helm.py` because the final helm yaml files were missing `apiVersion` key in some cases. Aditionally, the file names follow helm's convention & i've updated the component-chart to `0.9.1`.

Emply cluster on `dev`
empty-cluster

Helm install on `dev`
deploy-tot

Helm update on this branch (with seaweed-fs)
deploy-seaweed

File: packages/grid/devspace.yaml (hunk 1 above)
Comment (madhavajay) [reply packages/grid/devspace.yaml:272]: We can leave these to under `dev` only.

**Comment by madhavajay** [comment]
I will review this but, excellent work @yashgorana this is my favourite PR of the week! 🙌

**Review by madhavajay — APPROVED** [review]
Great work. I needed to add `localhost` to the default hostname for the changes to ingress when running in k3d, but other than that everything worked.

I imagine we might have some more to, do once we get the tests running again but for now this is ready for merge!

Definitely PR of the week! Keep it up. ❤️

**Pull request merged**